src/main/fc/settings.yaml

```diff
@@ -219,13 +219,13 @@ groups:
         default_value: 1000
         max: 9000
       - name: gyro_anti_aliasing_lpf_hz
-        description: "Gyro processing anti-aliasing filter cutoff frequency. In normal operation this filter setting should never be changed. In Hz"
-        default_value: 250
+        description: "Gyro processing anti-aliasing filter cutoff frequency. In normal operation this filter setting should never be changed. In samples"
+        default_value: 1
         field: gyro_anti_aliasing_lpf_hz
         max: 1000
       - name: gyro_main_lpf_hz
```

Comment:
This setting shold not be changed, but a new one like 'gyro_lulu_sample_count` added